src/ui/Accordion/Accordion.scss

```diff
@@ -0,0 +1,27 @@
+.Container {
+  display: inline-block;
+  width: 100%;
+  transition: all var(--transition-fast) ease-in-out var(--transition-faster);
+
+  &.Opened {
+    margin: var(--offset-small) 0;
+  }
+}
+
+.Header {
+  display: inline-block;
+  width: 100%;
+}
+
+.Content {
+  position: relative;
+  display: inline-block;
+  width: 100%;
+  overflow: hidden;
```

Comment:
Is overflow always hidden? It might prevent some elements to go beyond the outlines of Accordion content. For example some dropdown list with position absolute or some datepicker popup. 
`HeightTransitionBox` had the same issue, currently it has overflow `hidden` only during the transitioning phase